### PR TITLE
Fix delocalization path issue

### PR DESCRIPTION
**Pull request by Horneth**
This fixes the issue in PAPI2 and revert it to what it was before for PAPI1. Since this change was targeted at CWL and we don't plan to support CWL on PAPI1, it seems safer to revert it entirely in case it still breaks something else

### CODE REVIEW

File: src/bin/ci/resources/build_application.inc.conf [hunk 1]
```diff
@@ -1,3 +1,5 @@
+akka.http.host-connection-pool.max-open-requests = 1024
```
Comment (mcovarr) [inline src/bin/ci/resources/build_application.inc.conf]: stowaway?

File: centaur/src/main/resources/standardTestCases/custom_mount_point/custom_mount_point.wdl [hunk 2]
```diff
@@ -1,30 +1,35 @@
...
   }
 }
 
 workflow custom_mount_point {
-  call t {input: version="latest"}
+  call t {input: v="latest"}
 
   output {
-     t.out2
+     String o1 = t.out1
```
Comment (cjllanwarne) [inline centaur/src/main/resources/standardTestCases/custom_mount_point/custom_mount_point.wdl]: Can we also check these outputs in the `.test`:
```
metadata {
  "calls.custom_mount_point.t.backend": "Papi"
  "calls.custom_mount_point.t.backendStatus": "Success"

  "outputs.custom_mount_point.out1.": "bazqux"
  "outputs.custom_mount_point.out1.": "foobar"
}
```

File: src/bin/ci/resources/build_application.inc.conf (hunk 1 above)
Comment (cjllanwarne) [inline src/bin/ci/resources/build_application.inc.conf]: Is this related?

File: supportedBackends/google/pipelines/v2alpha1/src/main/scala/cromwell/backend/google/pipelines/v2alpha1/PipelinesApiAsyncBackendJobExecutionActor.scala [hunk 3]
```diff
@@ -86,4 +87,16 @@ class PipelinesApiAsyncBackendJobExecutionActor(standardParams: StandardAsyncExe
       })
     case _ => List.empty
   }
+
+  override def generateSingleFileOutputs(womFile: WomSingleFile, fileEvaluation: FileEvaluation) = {
```
Comment (cjllanwarne) [inline supportedBackends/google/pipelines/v2alpha1/src/main/scala/cromwell/backend/google/pipelines/v2alpha1/PipelinesApiAsyncBackendJobExecutionActor.scala]: Do we have a test for the original problem that this change was intended to fix when it was made?

Comment (Horneth) [reply supportedBackends/google/pipelines/v2alpha1/src/main/scala/cromwell/backend/google/pipelines/v2alpha1/PipelinesApiAsyncBackendJobExecutionActor.scala]: conformance test 55 (on PAPI2) is the test

File: src/bin/ci/resources/build_application.inc.conf (hunk 1 above)
Comment (Horneth) [reply src/bin/ci/resources/build_application.inc.conf]: No but all my PRs keep failing with this `max-open-requests` error so I'm trying to ~~sneak this in~~ fix it

File: centaur/src/main/resources/standardTestCases/custom_mount_point/custom_mount_point.wdl (hunk 2 above)
Comment (cjllanwarne) [reply centaur/src/main/resources/standardTestCases/custom_mount_point/custom_mount_point.wdl]: uh, oops - that's a typo, should be `out2` on the final line!
